.circleci/config.yml

```diff
@@ -311,16 +333,22 @@ workflows:
                 - /^main?/
                 - /^v\d+\.\d+\.\d+$/
                 - /^dep-update-[a-f0-9]{7}$/
+          context:
+            - dockerhub
       - run_premium_primitives_tests_on_main:
           name: "run premium primitives tests on main"
           filters:
             branches:
               only:
                 - main
+          context:
+            - dockerhub
       - run_premium_primitives_tests_on_release:
           name: "run premium primitives tests on release"
           filters:
             branches:
               ignore: /.*/
             tags:
               only: /^v\d+\.\d+\.\d+$/
+          context:
+            - dockerhub
```

Comment:
i think we can write this as `context: dockerhub`

Reply:
Fixed